plugin-server/src/cdp/schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod'
+
+export const CdpInternalEventSchema = z.object({
+    team_id: z.number(),
+    event: z.object({
+        uuid: z.string(),
+        event: z.string(),
+        // In this context distinct_id should be whatever we want to use if doing follow up things (like tracking a standard event)
+        distinct_id: z.string(),
+        properties: z.record(z.any()),
+        timestamp: z.string(),
+        url: z.string().optional(),
+    }),
```

Comment:
You probably have a much better grasp on this than me but are we "hacking the system" in any way by reusing the name _event_ here?

When pushing to Kafka we can absolutely use the `event` that triggered the error tracking issue to be created as the value for most of these properties but I'm guessing we should be using the issue itself instead. E.g. we should push something like:

```
...
event: z.object({
        uuid: issue.id,
        event: "$$error_tracking_issue_created",
        distinct_id: uuidv4(),
        properties: { name: issue.name, description: issue.description, event: processed_event },
        timestamp: issue.created_at,
        url: `https://us.posthog.com/error_tracking/${issue.id}`,
    }),
```

The `event` within `properties` is totally optional but maybe it'd be worth adding

Reply:
Yeah we are hacking it a little but sort of on purpose at the same time 😅 

So the "event" in our case is what internal thing happened. The `properties` should then contain the sub-info related to your exception event. So I think this is by design tbh.

There's some nuances here like the uuid should still be unique so that we can uniquely attribute _the event being fired_ not the issue ID for example.